src/lib/loaders/api/loader_without_authentication_factory.ts

```diff
@@ -127,38 +127,23 @@ export const apiLoaderWithoutAuthenticationFactory = <T = any>(
                       cached: true,
                     })
                   )
-                  // Trigger a cache update after returning the data.
-                  .then(
-                    tap(() =>
-                      deferThrottled(
-                        key,
-                        () =>
-                          callApi()
-                            .then(finish({ message: `Refreshing: ${key}` }))
-                            .then(reduceData)
-                            .then(cacheData)
-                            .catch(err => {
-                              if (err.statusCode === 404) {
-                                // Unpublished.
-                                cache.delete(key)
-                              }
-                            }),
-                        { requestThrottleMs: apiOptions.requestThrottleMs }
-                      )
-                    )
-                  )
                   // Cache miss.
-                  .catch(() =>
-                    callApi()
+                  .catch(() => {
+                    const cacheOptions: CacheOptions = {}
+                    if (apiOptions.requestThrottleMs) {
```

Comment:
Can you explain what this logic is doing?  Unclear as to what behavioral change removing `deferThrottled` and setting `cacheOptions.cacheTtlInSeconds = apiOptions.requestThrottleMs / 1000` will do here...

Reply:
I mentioned this in the PR description:

> We opted to use requetThrottleMs if present, since the intent of that (60 minutes for filters), is to essentially cache the data for that long. Otherwise we use 60 seconds.

This option was used to throttle backend refreshes...but thew 'intent', by developers, is essentially that this data is ok to be stale for that long. So in our new caching scheme we'll interpret this value as the cache TTL.

Reply:
Is there a reason to not rename this immediately?

Reply:
Not really! This was sort of a timeboxed experiment and so it was just easier to repurpose the variable here, vs the slightly bigger refactor + rename.

But that's definitely a _bit_ of an excuse...it's easy to just rename it (plus TS helps). There was no good reason other than that!

Reply:
Fair enough—just wondering if I’m missing some perhaps important details!